kernel/src/container.rs

```diff
@@ -1,8 +1,11 @@
+
+
```

Comment:
I have no idea. Now it let me delete them w/out re-adding them. I swear it added them before.  Fixed I guess.